### DISCUSSION TITLE
Add `closed` to `involves` filter or add separate `closed_by` filter

### DISCUSSION THREAD

**Discussion by manovotny** [community/community#68469] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

According to the [Search issues & PRs GitHub Docs](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-a-user-thats-involved-in-an-issue-or-pull-request): 

> You can use the `involves` qualifier to find issues that in some way involve a certain user. The `involves` qualifier is a logical OR between the `author`, `assignee`, `mentions`, and `commenter` qualifiers for a single user. In other words, this qualifier finds issues and pull requests that were either created by a certain user, assigned to that user, mention that user, or were commented on by that user.

I'd like to see `closed` be added to `involves` criteria. This, for example.

![CleanShot 2023-09-27 at 14 40 54@2x](https://github.com/community/community/assets/446260/27109f33-66ff-4400-8d18-6d8b467c1fde)

Where I didn't leave a comment or do anything else, but I closed the issue.

Or, if we could add a separate `closed_by` filter, that would work too.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by c-rhodes** [comment]
Also hit this, would be great to see this fixed.
